### PR TITLE
Convertor to convert List of Object to Json Array

### PR DESCRIPTION
**Pull request by hasankshaik**

### CODE REVIEW

File: common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	@Inject
+	ObjectMapper mapper;
+
+	@Inject
+	StringToJsonObjectConverter stringToJsonObjectConverter;
+
+	public JsonArray convert(final List<T> sourceList) {
+		JsonArrayBuilder jsonArrayBuilder = Json.createArrayBuilder();
+		if (sourceList == null) {
+			throw new ConverterException(String.format("Failed to convert %s to JsonObject", sourceList));
```
Comment (matt-rich) [inline common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java]: Error message should be updated as the conversion failed to a JsonArray, and below.

File: common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import javax.json.JsonArrayBuilder;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.gov.justice.services.common.converter.exception.ConverterException;
+
+/**
+ * Converts a List of Type T to JsonArray
+ */
+public class ListToJsonArraysConverter<T> implements Converter<List<T>, JsonArray> {
```
Comment (purple52) [inline common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java]: Should be Array (singular) in class name.

File: common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+	@Inject
+	ObjectMapper mapper;
+
+	@Inject
+	StringToJsonObjectConverter stringToJsonObjectConverter;
+
+	public JsonArray convert(final List<T> sourceList) {
+		JsonArrayBuilder jsonArrayBuilder = Json.createArrayBuilder();
```
Comment (purple52) [inline common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java]: Should be static import

File: common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java (hunk 1 above)
Comment (purple52) [inline common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java]: String.format should be static import

File: common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonArrayBuilder;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.gov.justice.services.common.converter.exception.ConverterException;
+
+/**
+ * Converts a List of Type T to JsonArray
```
Comment (purple52) [inline common/src/main/java/uk/gov/justice/services/common/converter/ListToJsonArraysConverter.java]: <T> should be defined in the javadoc

**Pull request closed without merging**